### ISSUE TITLE
MMM is installing bukkit version of mod when told to install fabric version

### ISSUE THREAD

**Issue by theofficialgman** [Senth/minecraft-mod-manager#202]
### Describe the bug 
I instruct mmm to install simple-voice-chat fabric mod with 
```
mmm -v 1.19.2 --mod-loader fabric --beta --alpha install simple-voice-chat
```
and it installs the bukkit version, not the fabric version
```
CurseForge has been disabled! See https://github.com/Senth/minecraft_mod_manager for more information.
simple-voice-chat
    🔍 Searching for mod
    No mod info found for voicechat-bukkit-1.19.2-2.3.3.jar
    🟢 Installed version bukkit-1.19.2-2.3.3
```

### Expected Behavior
install the latest beta/alpha (or release) fabric version:
https://modrinth.com/mod/simple-voice-chat/versions

### Info
 - OS: Bionic
 - Python --version: python 3.8
 - minecraft-mod-manager --version: 1.4.0

**Comment by Senth** [comment]
Hmm... I've been wrapping my head around this one. I can't seem to find the bug. It filters correctly on my system.

I also debugged the code checking so that everything got correctly filtered. The only reason I could see is that it wasn't tagged with any loaders first. This would cause mmm to treat it as any mod loader. A solution here would be to not treat empty `loaders` list as any mod loader.

Has the issue solved itself for you, or are you still having the issue?

**Comment by theofficialgman** [comment]
a newer release has been issued for the fabric version since the bukkit version came out.
that is why you can not replicate. you will need to modify the modrinth response to remove those newer versions and stop at 2.3.3 (or another version where the version for bukkit matches that for fabric)

here is an example, use minecraft 1.16.5, there is NO fabric version present at all, but there is a bukkit version. and MMM installs bukkit when told to install fabric. this should NOT be the case, it should error

```
mmm -v 1.16.5 --mod-loader fabric --beta --alpha install simple-voice-chat
CurseForge has been disabled! See https://github.com/Senth/minecraft_mod_manager for more information.
simple-voice-chat
    🔍 Searching for mod
    No mod info found for voicechat-bukkit-1.16.5-2.3.6.jar
    🟢 Installed version bukkit-1.16.5-2.3.6
```

**Comment by Senth** [comment]
I was a bit tired yesterday after work 😅 You're correct. I missed yesterday that quilt, bukkit, spigot, etc, was simply removed from the mod loaders list instead of set as unknown.

I added a test for the 1.16.5 version and tried install the fabric version, now it does not install anything which is correct 🙂 

Fixed in `1.4.2`

**Comment by theofficialgman** [comment]
no worries. thanks for your continued work on this software when you have the time 😄
